src/solver/src/grins_solver.C

```diff
@@ -52,6 +52,7 @@ namespace GRINS
       _max_linear_iterations( input("linear-nonlinear-solver/max_linear_iterations", 500 ) ),
       _continue_after_backtrack_failure( input("linear-nonlinear-solver/continue_after_backtrack_failure", false ) ),
       _continue_after_max_iterations( input("linear-nonlinear-solver/continue_after_max_iterations", false ) ),
+      require_residual_reduction( input("linear-nonlinear-solver/require_residual_reduction", false ) ),
```

Comment:
Sorry, I forgot to do compile and check the second time. Let me do that now.

On Fri, Sep 4, 2015 at 5:34 PM, Paul T. Bauman <notifications@github.com>
wrote:

> In src/solver/src/grins_solver.C
> <https://github.com/grinsfem/grins/pull/303#discussion_r38799877>:
>
> > @@ -52,6 +52,7 @@ namespace GRINS
> >        _max_linear_iterations( input("linear-nonlinear-solver/max_linear_iterations", 500 ) ),
> >        _continue_after_backtrack_failure( input("linear-nonlinear-solver/continue_after_backtrack_failure", false ) ),
> >        _continue_after_max_iterations( input("linear-nonlinear-solver/continue_after_max_iterations", false ) ),
> > +      require_residual_reduction( input("linear-nonlinear-solver/require_residual_reduction", false ) ),
>
> Did you try compiling this because I don't see how this could compile...
> This should be _require_residual_reduction (_ prefix)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grinsfem/grins/pull/303/files#r38799877>.
>



-- 
Vikram Garg
Postdoctoral Associate
Predictive Engineering and Computational Science (PECOS)
The University of Texas at Austin
http://web.mit.edu/vikramvg/www/

http://vikramvgarg.wordpress.com/
http://www.runforindia.org/runners/vikramg